### PR TITLE
Please Add mav-teleop-twist-keyboard package to be indexed in the rosdistro

### PR DESCRIPTION
**Pull request by Reka-Berci-Hajnovics**
# Please Add This Package to be indexed in the rosdistro.

melodic mav-teleop-twist-keyboard

# The source is here: 
https://github.com/dobots/mav_teleop_twist_keyboard

### CODE REVIEW

**Comment by Reka-Berci-Hajnovics** [comment]
Hi! Thank you for taking your time for running these tests. I was following the ros wiki on how to [release a package](http://wiki.ros.org/ROS/ReleasingAPackage). Point 2.2. "Adding a package to rosdistro" suggests to run the command `rosdistro reformat distribution.yaml` to fix any formatting errors. If I try to run this command I get the error message: `rosdistro: command not found. On [ros answers - how to run rosdistro command](https://answers.ros.org/question/100556/how-to-run-rosdistro-command/) it is stated that `rosdistro` is not used anymore. Could you please help with which command should I use to fix any formatting issues? Thank you in advance!

**Comment by nuclearsandwich** [comment]
> Could you please help with which command should I use to fix any formatting issues? Thank you in advance!

You can use the standalone `rosdistro_reformat` command which is part of the `python3-rosdistro` package. With this branch cloned locally you could run `rosdistro_reformat index-v4.yaml` to update distribution files that require reformatting.

**Comment by Reka-Berci-Hajnovics** [comment]
Thank you for your answer!  I have found another tutorial for releasing a package, which is based on the tool called bloom: http://wiki.ros.org/bloom/Tutorials/FirstTimeRelease

In the last step of the tutorial, it notifies the build farm and creates a pull request so there is no need to edit the distribution.yaml file manually. I'll try to use that tool first. If I run into problems, I'll try your solution :)

File: melodic/distribution.yaml [hunk 1]
```diff
@@ -6149,6 +6149,12 @@ repositories:
       url: https://github.com/ethz-asl/mav_comm.git
       version: master
     status: developed
+  mav_teleop_twist_keyboard:
+    source:
+      type: git
+      url: https://github.com/dobots/mav_teleop_twist_keyboard.git
+      version: master
+    status: maintained
```
Comment (mabelzhang) [inline melodic/distribution.yaml:6157]: It looks like you just have trailing spaces after the `status: maintained` line.
```suggestion
    status: maintained
```

Comment (Reka-Berci-Hajnovics) [reply melodic/distribution.yaml]: Thanks for the info, I've removed the white spaces.

**Review by mabelzhang — COMMENTED** [review]
Looks like you don't have a LICENSE file. Please see the [contributing guide](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md) about the license.

The check is failing but a merge from `master` should fix that.

File: melodic/distribution.yaml [hunk 2]
```diff
@@ -6149,6 +6149,12 @@ repositories:
       url: https://github.com/ethz-asl/mav_comm.git
       version: master
     status: developed
+  mav_teleop_twist_keyboard:
+    source:
```
Comment (ivanpauno) [inline melodic/distribution.yaml:6153]: Consider also adding a `doc` entry

**Review by ivanpauno — APPROVED** [review]
LGTM!

**Pull request merged**